src/H5Glink.c

```diff
@@ -175,70 +175,6 @@ H5G__link_cmp_corder_dec(const void *lnk1, const void *lnk2)
     FUNC_LEAVE_NOAPI(ret_value)
 } /* end H5G__link_cmp_corder_dec() */
 
-/*-------------------------------------------------------------------------
- * Function:	H5G__ent_to_link
- *
- * Purpose:     Convert a symbol table entry to a link
- *
- * Return:	Non-negative on success/Negative on failure
- *
- *-------------------------------------------------------------------------
- */
-herr_t
-H5G__ent_to_link(H5O_link_t *lnk, const H5HL_t *heap, const H5G_entry_t *ent, const char *name)
```

Comment:
This function appears to have been moved to `H5Gent.c`, while `H5G__ent_convert()` from `H5Gent.c` was renamed to `H5G__link_to_ent()` and moved into this file.

Reply:
Yes, made more senes with what the routines were doing